File: csharp/MaxPainInfrastructure/Models/Daily.cs
namespace MaxPainInfrastructure.Models
{

    public class Daily
    {
        public string? Ticker { get; set; }
        public string? Source { get; set; }
        public DateTime? Date { get; set; }
        public double? Open { get; set; }
        public double? High { get; set; }
        public double? Low { get; set; }
        public double? Close { get; set; }
        public double? AdjClose { get; set; }
        public long Volume { get; set; }
        public double? Dividends { get; set; }
        public double? StockSplits { get; set; }
    }
}

File: csharp/MaxPainUI/ClientApp/src/app/models/daily.ts
export class Daily {
    ticker: string;
    source: string;
    date: Date;
    open: number;
    high: number;
    low: number; 
    close: number;
    adjClose: number;
    volume: number;

    constructor(jsonObj?: any) {
        if (jsonObj) {
            Object.keys(jsonObj).forEach(key => { this[key] = jsonObj[key]; });
        }
    }
} 
